core/domain/state_domain.py

```diff
@@ -3341,33 +2594,14 @@ def validate(
                 'Expected card_is_checkpoint to be a boolean, '
                 'received %s' % self.card_is_checkpoint)
 
-        self.written_translations.validate(content_id_list)
-        self.recorded_voiceovers.validate(content_id_list)
+        self.recorded_voiceovers.validate(self.get_translatable_content_ids())
```

Comment:
Will we also be introducing the new structure for voiceovers later?

Reply:
Voiceovers are not expected to be big i.e, their contents are expected to be a limited length (fixed length of the filename), so there isn't any plan as of now!

Reply:
Ah, okay, I didn't realize that voiceover files are stored separately.